skypy/power_spectrum/_camb.py

```diff
@@ -29,6 +30,8 @@ def camb(wavenumber, redshift, cosmology, A_s, n_s):
     n_s : float
```

Comment:
Does it make sense to make some changes to the docstring in this PR?

Reply:
Can you say a little bit more what you mean about this?

Reply:
I don't see any problem other than the units for the wavenumber should be left as `Mpc-1` not changed to `[Mpc^-1]` for consistency with #272

Reply:
Agreed with Richard